asciimatics/screen.py

```diff
@@ -1914,6 +1914,7 @@ def close(self, restore=True):
                 curses.echo()
                 curses.nocbreak()
                 curses.endwin()
+                signal.signal(signal.SIGWINCH, signal.SIG_DFL)
```

Comment:
Also, shouldn't we do the same for the ctrl-c and ctrl-z signal handlers?  I can't imagine it's good for them to be left lying around after you've dropped out of the curses screen...